src/ui/common/src/components/integrations/dialogs/airflowDialog.tsx

```diff
@@ -28,12 +28,15 @@ export const AirflowDialog: React.FC<Props> = ({
   return (
     <Box sx={{ mt: 2 }}>
       <IntegrationTextInputField
+        name="host"
         spellCheck={false}
         required={true}
         label="Host *"
         description="The hostname of the Airflow server."
         placeholder={Placeholders.host}
         onChange={(event) => {
+          // TODO: Send this new updated value to react-hook-form's context.
+          // NOTE: Cannot have url prefix in here.
```

Comment:
TODO: Use register and setValue here to get the logic for removing the http:// or https:// prefix from the url.

Reply:
resolved.